examples/qtmegachatapi/contactItemWidget.cpp

```diff
@@ -4,23 +4,21 @@
 #include <QMessageBox>
 #include <QMenu>
 
-ContactItemWidget::ContactItemWidget(QWidget *parent , megachat::MegaChatApi * megaChatApi, mega::MegaApi * megaApi, megachat::MegaChatHandle userHandle) :
+ContactItemWidget::ContactItemWidget(QWidget *parent , megachat::MegaChatApi *megaChatApi, mega::MegaApi *megaApi, mega::MegaUser *contact) :
     QWidget(parent),
     ui(new Ui::ChatItem)
 {
     mMegaApi = megaApi;
     mMegaChatApi = megaChatApi;
-    mUserHandle = userHandle;
-    const char *contactEmail = megaChatApi->getContactEmail(userHandle);
+    mUserHandle = contact->getHandle();
+    const char *contactEmail = contact->getEmail();
```

Comment:
Check the ownership of `MegaUser::getEmail()`. It's not yours, and you are storing a pointer to it that will eventually become invalid.